components/MyMessage.vue

```diff
@@ -189,14 +189,24 @@
                   message.location &&
                   message.item
                 "
-                variant="secondary"
-                disabled
-                class="mr-2 mb-1"
-                title="You will be able to repost this soon"
+                class="position-relative"
               >
-                <v-icon class="d-none d-sm-inline" icon="sync" /> Repost
-                <span class="small">{{ timeago(message.repostat) }}</span>
-              </b-button>
+                <b-button
+                  variant="secondary"
+                  class="mr-2 mb-1"
+                  title="You will be able to repost this soon"
+                  @click.stop="repostWhenUnavailable"
+                >
+                  <v-icon class="d-none d-sm-inline" icon="sync" /> Repost
+                  <span class="small">{{ timeago(message.repostat) }}</span>
+                </b-button>
+                <p
+                  class="invalid-feedback position-absolute bg-white text-center"
+                  :style="{ display: triedToRepost ? 'block' : 'hidden' }"
```

Comment:
I try to avoid setting style values directly (I think it's a code smell).  Please can you switch to use the `:class=` variant and then set `d-none` / `d-block` classes using this flag?

Reply:
Done. Forgot about these Bootstrap's classes, sorry.